JS9/JS-9-1.js

```diff
@@ -0,0 +1,130 @@
+function newTask() {
+  console.log("========================New Task===========================");
+}
+//1. Создайте класс Employee с полями:
+//firstName — имя сотрудника (строка).
+//lastName — фамилия сотрудника (строка).
+//profession — профессия сотрудника (строка).
+//Приватное поле salary — зарплата сотрудника (число).
+class Employee {
+  constructor(firstName, lastName, profession, salary) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.profession = profession;
+    this._salary = salary;
+  }
```

Comment:
```suggestion
class Employee {
  #salary // because it should be private property according to task description
  constructor(firstName, lastName, profession, salary) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.profession = profession;
    this.#salary = salary;
  }
```